conf.py

```diff
@@ -104,6 +105,16 @@
 
 todo_include_todos = True
 
+# sphinxext-opengraph config
+ogp_site_url = "https://devguide.python.org/"
+ogp_site_name = "Python Developer's Guide"
+ogp_image = "_static/og-image.png"
+ogp_custom_meta_tags = [
+    '<meta property="og:image:width" content="1200">',
+    '<meta property="og:image:height" content="630">',
+    '<meta name="theme-color" content="#3776ab" />',
```

Comment:
FWIW I looked into using a gradient that matches both colors of the Python logo, but this doesn't seem supported.